security/checker.py

```diff
@@ -51,8 +51,10 @@ def configure(self, config):
     def _discover(self, plugin, region_name):
         logging.info("Discovering issues by plugin %s", plugin)
```

Comment:
just a nit: There would now be 2 lines about discovering by region and by plugin. I'd rather add info on region to this info, than add a debug logging